.tekton/integration-tests/lightspeed-console-pre-commit-417.yaml

```diff
@@ -201,31 +205,27 @@ spec:
               export CYPRESS_LOGIN_PASSWORD=$(cat ${PASSWORD_PATH})
               echo "${CYPRESS_LOGIN_PASSWORD}"
               echo "---------------------------------------------"
-              dnf -y update && \
-                  dnf -y install golang wget git awk
+              apt-get update \
+                  && apt-get install curl
```

Comment:
I don't see curl being used anywhere, so maybe we don't need this now?

Reply:
yeah, we don't. I replaced all instances with wget because it's installed by default in the cypress package. Then I got curl back but never changed any of the instances. We can remove it